governance-charter.md

```diff
@@ -51,6 +53,50 @@ following:
 This work is to be handled by the Governance Committee or delegated to other
 project groups like the [Technical Committee](./tech-committee-charter.md).
 
+### Project Management
+
+#### Develop project proposals
+Efforts and initiatives that are too large, detailed, or important to be
+resolved in an issue or pull request are referred to as *projects*. These
+larger efforts often require collaboration between subject matter experts,
+the Technical Committee, and OpenTelemetry community members, and involve
+the creation of either a temporary or permanent SIG to accomplish their goals.
+The Governance Committee is responsible for reviewing project proposals and
+ensuring  to accomplish their goals. The [project management](https://github.com/open-telemetry/community/blob/main/project-management.md)
```

Comment:
```suggestion
ensuring they accomplish their goals. The [project management](https://github.com/open-telemetry/community/blob/main/project-management.md)
```

Reply:
This whole line is actually a duplicate of the next paragraph, which has better wording. I've removed it.